src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -176,6 +184,11 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                .WithDescription("If provided, a path to the file that will be used to write the output when using --enable-json-output.")
                .Callback(jsonOutputFile => JsonOutputFile = jsonOutputFile);
 
+            Parser
+               .Setup<string>("runtime")
```

Comment:
Which runtime?

Reply:
The host runtime. We decided to have that be the parameter that users can pass in